docs/disaster-recovery.md

```diff
@@ -0,0 +1,443 @@
+> it should be created in the same namespace as of PVC.
+
+`replicationState` is the state of the volume being referenced.
+ Possible values are primary, secondary, and resync.
+
+* `primary` denotes that the volume is primary.
+* `secondary` denotes that the volume is secondary.
+* `resync` denotes that the volume needs to be resynced.
+
+To check VolumeReplication CR status:
+
+```bash
+kubectl get volumereplication pvc-volumereplication  --context=cluster-1 -oyaml
+```
+
+>```yaml
+>...
+>spec:
+>  dataSource:
+>    apiGroup: ""
+>    kind: PersistentVolumeClaim
+>    name: rbd-pvc
+>  replicationState: primary
+>  volumeReplicationClass: rbd-volumereplicationclass
+>status:
+>  conditions:
+>  - lastTransitionTime: "2021-05-04T07:39:00Z"
+>    message: ""
+>    observedGeneration: 1
+>    reason: Promoted
+>    status: "True"
+>    type: Completed
+>  - lastTransitionTime: "2021-05-04T07:39:00Z"
+>    message: ""
+>    observedGeneration: 1
+>    reason: Healthy
+>    status: "False"
+>    type: Degraded
+>  - lastTransitionTime: "2021-05-04T07:39:00Z"
+>    message: ""
+>    observedGeneration: 1
+>    reason: NotResyncing
+>    status: "False"
+>    type: Resyncing
+>  lastCompletionTime: "2021-05-04T07:39:00Z"
+>  lastStartTime: "2021-05-04T07:38:59Z"
+>  message: volume is marked primary
+>  observedGeneration: 1
+>  state: Primary
+>```
+
+* Take a backup of PVC and PV object on primary cluster(cluster-1)
+
+  * Take backup of the PVC `rbd-pvc`
+
+    ```bash
+    kubectl get pvc rbd-pvc -oyaml >pvc-backup.yaml
+    ```
+
+  * Take a backup of the PV, corresponding to the PVC
+
+    ```bash
+    kubectl get pv/pvc-65dc0aac-5e15-4474-90f4-7a3532c621ec -oyaml >pv_backup.yaml
+    ```
+
+>:bulb: We can also take backup using external tools like **Velero**.
+> Refer [velero documentation]((https://velero.io/docs/main/)) for more information.
+
+* Restoring on the secondary cluster(cluster-2)
+
+  * Create storageclass on the secondary cluster
+
+  ```bash
+    kubectl create -f examples/rbd/storageclass.yaml --context=cluster-2
+  ```
+
+  > ```bash
+  > storageclass.storage.k8s.io/csi-rbd-sc created
+  > ```
+
+  * Create VolumeReplicationClass on the secondary cluster
+
+  ```bash
+        cat <<EOF | kubectl --context=cluster-2 apply -f -
+        apiVersion: replication.storage.openshift.io/v1alpha1
+        kind: VolumeReplicationClass
+        metadata:
+          name: rbd-volumereplicationclass
+        spec:
+          provisioner: rbd.csi.ceph.com
+          parameters:
+            mirroringMode: snapshot
+            replication.storage.openshift.io/replication-secret-name: csi-rbd-secret
+            replication.storage.openshift.io/replication-secret-namespace: default
+        EOF
+  ```
+
+  > ```bash
+  > volumereplicationclass.replication.storage.openshift.io/rbd-volumereplicationclass created
+  > ```
+
+  * Remove the `claimRef` on the backed up PV objects in yaml files
+
+  ```yaml
```

Comment:
Add note why this need to be removed and when user need to remove this like (when manually coping the PV and PVC objects to second cluster)